# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: uriloader/preload/moz.build
# Commit: 86c478d114ec
# Full Hash: 86c478d114ec223ecf3698ddbb52977bf786f10c
# Author: Honza Bambas <honzab.moz@firemni.cz>
# Date: 2020-05-11 21:47:06
# Regressor Bug: 1618287
# File Overlap Count: 1
# Description:
#   Bug 1618287 - PreloaderBase class, r=smaug
#   
#   Depends on D71800
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D67479
# ==============================================================================

diff -r 5c64c1dfb318 -r 86c478d114ec uriloader/preload/moz.build
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/uriloader/preload/moz.build	Mon May 11 14:20:39 2020 +0000
@@ -0,0 +1,20 @@
+# -*- Mode: python; indent-tabs-mode: nil; tab-width: 40 -*-
+# vim: set filetype=python:
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+with Files("**"):
+    BUG_COMPONENT = ("Core", "Networking")
+
+EXPORTS.mozilla += [
+    'PreloaderBase.h',
+    'PreloadHashKey.h',
+]
+
+UNIFIED_SOURCES += [
+    'PreloaderBase.cpp',
+    'PreloadHashKey.cpp',
+]
+
+FINAL_LIBRARY = 'xul'
